test/Conversion/amd/builtin_func_to_llvm.mlir

```diff
@@ -10,3 +11,12 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 1 : i32, ttg.targ
     tt.return
   }
 }
+
+module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 1 : i32, ttg.target = "hip:gfx942", "ttg.threads-per-warp" = 64 : i32} {
+  tt.func public @test_fast_tanhf(%arg0: tensor<64xf32, #blocked>) {
```

Comment:
Can you add `// CHECK-LABEL: test_fast_tanhf`? Please also do the same for the above test so that we can better isolate them.